src/rules/at-extend-no-missing-placeholder/index.js

```diff
@@ -1,6 +1,7 @@
 import { utils } from "stylelint"
+import { namespace } from "../../utils"
 
-export const ruleName = "at-extend-no-missing-placeholder"
+export const ruleName = namespace("at-extend-no-missing-placeholder")
```

Comment:
I think we should update the *Contributing* section of the docs to require namespacing names of new rules, and doing it exactly this way (no hardcoded namespace, no other utils, etc.).